### PR TITLE
Clean up Astrea launch system

### PR DESCRIPTION
**Pull request by agonyG4**
### Motivation
- Remove user-specific hardcodes and inline QML/Python/ shell fragments introduced by prior experiments and make the runtime portable via `ASTREA_ROOT`, XDG dirs, or `HOME`.
- Harden the app launch/Exec parsing and add a safer internal argv-based launch request API for trusted internal calls while keeping `sh -lc` only for explicit user command strings.
- Move Spotlight/Performance QML inlined helpers to small backend helpers so UIs stay thin and easier to test.
- Improve install and latency tooling to fail loudly on missing pieces and add a smoke test for the latency boost lifecycle.

### Description
- Added an `Argv` launch request variant and plumbing to accept argv+working_dir trusted launches, and kept `--command` as `sh -lc` for untrusted user strings (`src/System/launch/src/lib.rs`).
- Fixed `Exec` parsing edge cases in `parse_exec_line` so field codes like `%f`, `"%u"`, escaped spaces, `%%`, and `%c` do not discard the current token; added unit tests covering these cases and serialized `Argv` requests (`src/System/launch/tests/launcher_tests.rs`).
- Implemented two small CLI helpers `astrea-performance` and `astrea-spotlight` to replace bulky inline QML/`python -c` blocks and updated QML to call these backends (`src/System/scripts/astrea-performance`, `src/System/scripts/astrea-spotlight`, `src/Apps/Settings/pages/system/Performance.qml`, `src/Quickshell/spotlight/Spotlight.qml`).
- Fixed `astrea_statusd.py` by adding the missing `import os` and made `ASTREA_ROOT` resolver use the environment fallback (`src/System/services/astrea_statusd.py`).
- Pulled the Bluetooth QML implementation out of the network module into `modules/bluetooth` and updated the bar to import it directly, keeping `modules/network` focused on `NetworkProcess` (`src/Quickshell/bar/modules/bluetooth/BluetoothProcess.qml`, `src/Quickshell/bar/Bar.qml`, `src/Quickshell/bar/modules/network/qmldir`).
- Cleaned install/service scripts to use a generic `astrea_bin_dir`/`ASTREA_ROOT`, removed hardcoded `agony` fallbacks, validated helper files before installing sudoers, and made sudoers install fail if `visudo` rejects it (`src/System/services/astrea-services.sh`, `src/System/services/install-latency-burst-helper.sh`).
- Added formatting and minor refactors (shell/python/rust formatting) and a smoke integration script that verifies `astrea-launch doctor`, `--desktop` launch flow, latency boost activation, daemon termination and rollback persisted to history (`src/System/services/smoke-launch-latency.sh`).

### Testing
- Ran `cargo test --manifest-path src/System/launch/Cargo.toml` and all launcher unit tests passed (`6 tests` passed).
- Compiled Python services and scripts with `python3 -m py_compile` for `astrea_statusd.py`, `astrea_latencyd.py`, `astrea-performance`, and `astrea-spotlight`, which succeeded.
- Performed shell syntax checks with `bash -n` on the services and used `shfmt`/`black` where available; checks passed and files were formatted.
- Executed the latency smoke script `src/System/services/smoke-launch-latency.sh`, which exercised `astrea-launch doctor`, a desktop launch, started `astrea-latencyd`, sent a boost, verified active state, killed the daemon and confirmed rollback events in history; the script completed successfully in CI container.
- Note: `qmlformat` was not available in the container so QML formatting was not run here (warning only).

------
[Codex Task](https://chatgpt.com/codex/cloud/tasks/task_e_6a05adff1b348326a1865b232ef965b9)

### CODE REVIEW

File: src/System/launch/src/lib.rs [hunk 1]
```diff
@@ -221,7 +237,8 @@ pub fn run_launch(request: LaunchRequest) -> Result<LaunchRecord, String> {
 pub fn run_launch_via_daemon(request: LaunchRequest) -> Result<LaunchRecord, String> {
     match send_launch_request(&request) {
         Ok(record) => Ok(record),
-        Err(_) => run_launch(request),
+        Err(LaunchdError::Connect) => run_launch(request),
+        Err(LaunchdError::Request(err)) => Err(err),
```
[the remark this answers is not in the document]
Comment (agonyG4) [reply src/System/launch/src/lib.rs:241]: @codex you can do all these changes
